src/compiler/abepi.rs

```diff
@@ -13,29 +13,31 @@ use crate::{
 
 #[derive(Debug)]
 /// Result of compiling an arbitrary boolean expression into a PI
-pub struct CompilationResult<F, V> {
+pub struct CompilationResult<F, V, M> {
     #[allow(dead_code)]
     pub dsym: DebugSymRef,
     // 0 is true, !=0 is false
-    pub anti_booly: Expr<F, V>,
+    pub anti_booly: Expr<F, V, M>,
```

Comment:
I think we talked about this, here it would be `pub anti_booly: Expr<F, V, ()>` so there is no need to add M to CompilationResult. And it is true for the rest, Everywhere that Expr is used M is set to `()`.

Reply:
I'm sorry, I was just thinking in moving forward on applying already `M` where I was thinking it would be necessary.
Fixing it